lib/fizzy/execute.cpp

```diff
@@ -362,6 +375,25 @@ inline double fneg(double value) noexcept
     return bit_cast<double>(bit_cast<uint64_t>(value) ^ F64SignMask);
 }
 
+template <typename T>
+T fcopysign(T a, T b) noexcept = delete;
+
+template <>
+inline float fcopysign(float a, float b) noexcept
```

Comment:
Here we have small issue with the name: `fcopysign` vs `std::copysign` and `signbit` vs `std::signbit`.

Reply:
Can rename `fcopysign` to `copysign`, it was an already existing feature and just moved. However if it is 100% compatible with `std::copysign` then should be renamed.

Reply:
Hmm actually https://en.cppreference.com/w/cpp/numeric/math/copysign has some interesting comment:
> float       copysign ( float mag, float sgn );
> 
> If mag is NaN, then NaN with the sign of sgn is returned.
> **If sgn is -0, the result is only negative if the implementation supports the signed zero consistently in arithmetic operations.**

Reply:
I think it is fine to rename to `copysign` and follow C std names (which are bad by themselves).

The C standard does not require IEEE 754, but WebAssembly does. So we must use compilers that support IEEE 754. E.g. GCC supports signed zeros by default, but this can be disabled with `-fno-signed-zeros`. https://gcc.gnu.org/wiki/FloatingPointMath